### PR TITLE
Add rules for adding new SDKs

### PR DESCRIPTION
**Pull request by duglin**
Per the rules, I think this requires a 1 week vote.

All SDK maintainers can vote.

Vote closes on Monday Mar 8, 3:30pm ET

/cc @cloudevents/sdk-maintainers 

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: community/SDK-GOVERNANCE.md [hunk 1]
```diff
@@ -8,6 +8,27 @@ The community is organized as follows:
   [cloudevents/sdk-go project](https://github.com/cloudevents/sdk-go)
 - The union of all the `sdk-*-maintainers` assembles the _sdk maintainers_ group
 
+## New SDKs
+
+To propose a new SDK for the community, a PR should be opened in the `spec`
+repository with the documentation changes necessary to point to the new repo.
+For example, changes to the `README.md` and `SDK.md` files. At that time the
```
Comment (slinkydeveloper) [inline community/SDK-GOVERNANCE.md:15]: Can we add a sentence here about the "usefulness" of the new sdk, like "new sdk should integrate with a technology stack that actual sdk portfolio doesn't reach", or something like that

Comment (duglin) [reply community/SDK-GOVERNANCE.md]: done - see what you think

Comment (slinkydeveloper) [reply community/SDK-GOVERNANCE.md]: Ok for me

**Comment by duglin** [comment]
Vote is now closed.
Yes: @slinkydeveloper @n3wscott @lholmquist @jskeet 
No: none
Vote passes - will merge.

**Pull request merged**